package.json

```diff
@@ -25,20 +25,20 @@
     "url": "https://github.com/dLobatog/npm2rpm/issues"
   },
   "dependencies": {
-    "async": "~1.5.2",
-    "colors": "~1.1.2",
-    "commander": "~2.9.0",
+    "async": "~2.6.1",
+    "colors": "~1.3.0",
+    "commander": "~2.15.1",
     "datejs": "*",
     "handlebars": "^4.0.11",
     "normalize-package-data": "^2.4.0",
     "npm-remote-ls": "~1.3.2",
-    "request": "~2.81.0",
-    "semver": "^5.4.1",
-    "tar": "~2.2.1",
-    "tmp": "~0.0.28"
+    "request": "~2.87.0",
+    "semver": "^5.5.0",
```

Comment:
This change isn't needed. We work with `^5.4.1` and that allows the same max versions as `^5.5.0`.